### ISSUE TITLE
Gameplay: Complete Round: Post to Facebook 

### ISSUE THREAD

**Issue by robinneb** [robinneb/golfzing#79]
User story: As a user I want to post my gameplay results to Facebook. 

Specifications -

**Comment by robinneb** [comment]
@digitalbrandgroup - Requires Jeremiah's review please. Could use Dev input specifically to the imageURL proposal in the story above. Tx!

**Comment by digitalbrandgroup** [comment]
This is a question for our engineers to see if the type of image you want snapshotted and sent to FB will be possible. We will find out as we start putting this screen together…

Ruby: Please check with our delivery team if there are any feasibility issues with the story described here.

Jeremiah

**Comment by digitalbrandgroup** [comment]
@robinneb 
Can you clarify what you meant by "an image of the Round"? is that from igolf? can you maybe provide a sample?

**Comment by robinneb** [comment]
This screen (Attached) minus any buttons or navigation.

**Comment by digitalbrandgroup** [comment]
@robinneb 
I don't think the image attachment came through but is this the image you want to be posted on FB?
![screen shot 2017-02-20 at 8 57 03 am](https://cloud.githubusercontent.com/assets/376210/23134742/c16e2062-f74a-11e6-8a09-72adf193c43f.png)

**Comment by robinneb** [comment]
Yes - sorry it did not come through but yes that image minus all of the
bottom navigation and the buttons - just the stats.

**Comment by digitalbrandgroup** [comment]
sounds good, thanks!

![screen shot 2017-02-20 at 9 21 10 am](https://cloud.githubusercontent.com/assets/376210/23135560/022131d2-f74e-11e6-9c22-5144e206dd21.png)
